doc/nrf/ug_thingy91.rst

```diff
@@ -205,35 +205,11 @@ Complete the following steps after installing the |nRFVSC|:
 Building and programming on the command line
 ============================================
 
-Complete the :ref:`command-line build setup <build_environment_cli>` before you start building |NCS| projects on the command line.
+.. |cmd_folder_path| replace:: on the nRF9160 SiP component and ``ncs/nrf/applications/connectivity_bridge`` when building the source code for the :ref:`connectivity_bridge` application on the nRF52840 SoC component
 
-To build and program the source code from the command line, complete the following steps:
+.. |cmd_build_target| replace:: ``thingy91_nrf9160`` or ``thingy91_nrf9160_ns`` if building for the nRF9160 SiP component and ``thingy91_nrf52840`` if building for the nRF52840 SoC component
```

Comment:
`thingy91_nrf9160` should probably be removed as well.

Reply:
There are some sample that uses this build target. So, I would keep this build target.

Reply:
Then those samples should be updated to only use `_ns`. Unless it's a sample for a secure domain application.